inputs_supply.md

```diff
@@ -183,30 +182,58 @@ Note that for the Shared Socioeconomic Pathways (SSPs), different inputs are use
+GCAM uses producer prices to initialize the model (future prices are endogenous). Those prices are provide in [GCAMDATA_FAOSTAT_ProducerPrice_170Regs_185PrimaryItems_2010to2020.csv.gz](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/aglu/FAO/GCAMDATA_FAOSTAT_ProducerPrice_170Regs_185PrimaryItems_2010to2020.csv.gz).
 
 
 ##### Cost of production
 
-The costs associated with land, irrigation, and fertilizer are endogenously determined in GCAM (see [Land Supply](supply_land.html)). Other costs of production are exogenously specified and the data used for those costs can be found in [USDA_cost_data.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/aglu/USDA_cost_data.csv), with [USDA_item_cost.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/aglu/USDA_item_cost.csv) specifying which costs are included in GCAM. Note that we use cost information for the USA in all regions.
+The costs associated with land, irrigation, and fertilizer are endogenously determined in GCAM (see [Land Supply](supply_land.html)). Other costs of production are exogenously specified and the data used for those costs can be found in [USDA_cost_data.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/aglu/USDA/USDA_cost_data.csv), with [USDA_item_cost.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/aglu/USDA/USDA_item_cost.csv) specifying which costs are included in GCAM. Note that we use cost information for the USA in all regions.
 
 ##### Emissions 
 
 Historical non-CO<sub>2</sub> emissions information is provided in the GCAM release as "pre-built" data aggregated to GCAM regions, technologies, and fuels. Users that want to build using CEDS raw data, for example to build for different regional aggregations, will need to generate CEDS data using the open-source [CEDS system](https://github.com/JGCRI/CEDS) and place the resulting emissions data by country, fuel, and sector within the [CEDS folder](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/emissions/CEDS).
 
 
+
+#### gcamdata-faostat
+
+There has been a significant data method update for the GCAM AgLU (Agriculture and Land Use) sectors in GCAM v7, which is described in detail in [Core Model Proposal #360: GCAM agriculture and land use (AgLU) data and method updates: connecting land hectares to food calories](cmp/CMP_360-AgLU_data_method_updates.pdf). An additional package is developed to facilitate the processing of data from FAOSTAT in a transparent, traceable, and consistent manner. This update encompasses the following key advancements:
+
+1. Functions have been developed to automate the updating of AgLU data, primarily sourced from FAOSTAT. This automation facilitates future base year updates and allows for other data adjustments to be easily incorporated. Note that most of the data in `aglu/FAO` folder are now generated by gcamdata-faostat.
```

Comment:
I'm not sure if "base year updates" is a common term.  I suggest replacing with "updates to the model base year", "updates to advance the model's final historical period", or something like that (slightly more descriptive).

Reply:
"in `aglu/FAO` folder" should be "in the `aglu/FAO` folder"

Reply:
Both updated. Thanks!